ui/src/serviceWorker.js

```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
```

Comment:
Not sure we need this one for now.

Reply:
no need for offline capability for us. Would be cool though :-)

Reply:
It is backbone anyway and gives application shel for production builds only.
Removing this may affect production builds that will look for serviceWorker in root folder.
It should be removed with manifest.